### ISSUE TITLE
Question: how can one record f-key strokes

### ISSUE THREAD

**Issue by sporkisfaster** [joao-carloto/ChromeRobot#5]
I have a need to record function key strokes for test suites and this extension doesn’t seem to register them. Am I not doing something properly or is this not supported

**Comment by joao-carloto** [comment]
Yes. Apparently the "Press key" option is not working for the Fn key. Still works for other keys (letters, numbers, enter, etc). I probably won't have time to check if this has a solution in the near future.
